cpp/monitoring/gauge.h

```diff
@@ -25,12 +25,13 @@ class Gauge : public Metric {
 
   void Set(const LabelTypes&... labels, double value);
 
+  // TODO(alcutter): Not over the moon about having this here.
```

Comment:
Yeah, I don't really have a plan at the moment.
It won't actually become a problem until the day that we try to add things like Histogram support, which could quite possibly be Friday the 15th of Never.